inst/rmarkdown/templates/wellcomeor/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,154 @@
+---
+title: Wellcome Open Research Article Template
+titlenote: The title should be detailed enough for someone to know whether the article would be of interest to them, but also concise. Please ensure the broadness and claims within the title are appropriate to the content of the article itself.
+author:
+  - name: Author Name1
+    affil: Address1
+  - name: Author Name2
+    affil: Address2
+abstract: |
+  Abstracts should be up to 300 words and provide a succinct summary of the article. Although the abstract should explain why the article might be interesting, care should be taken not to inappropriately over-emphasise the importance of the work described in the article. Citations should not be used in the abstract, and the use of abbreviations should be minimized. If you are writing a Research or Systematic Review article, please structure your abstract into Background, Methods, Results, and Conclusions.
+keywords: |
+  Please list up to eight keywords to help readers interested in your article find it more easily.
+
+date: "`r Sys.Date()`"
+bibliography: sample.bib
+csl: elsevier-vancouver.csl
```

Comment:
Could we suggest here to use the URL for this CLS, and add a comment to indicate to download and use local if one prefer ? This would be a way for us to avoid including the CSL file and having a potential outdated one in the package

```suggestion
# You can download the csl file if preferred and use it locally with `csl: elsevier-vancouver.csl`
csl: https://www.zotero.org/styles/elsevier-vancouver
```